### PR TITLE
Feature/141 conversation reconnect after 24 hours

### PR DESCRIPTION
**Pull request by wayangalihpratama**
- [x] Add conversation reconnect template env into `akvo-config`
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208848867296176

### CODE REVIEW

File: backend/core/socketio_config.py [hunk 1]
```diff
@@ -246,6 +196,64 @@ async def save_chat_history(
...
+
+    if content_sid and not TESTING:
+        template_content = get_template_content_from_json(
+            content_sid=content_sid
+        )
+        if template_content:
+            conversation_reconnect_message = template_content.replace(
+                "{{1}}", client_name
+            ).replace("{{2}}", clean_message_body)
+            await twilio_client.whatsapp_message_template_create(
```
[the remark this answers is not in the document]
Comment (jcbashdown) [reply backend/core/socketio_config.py:248]: It looks like it sends the reconnect template to the farmer but shouldn't it also take `clean_message_body` as a `content_variable` in that case?

**Review by jcbashdown — CHANGES_REQUESTED** [review]
Looks good! Mostly there I think. Just highlighted a couple of things I find unclear and asked a couple of questions.

File: backend/db/crud_chat.py [hunk 2]
```diff
@@ -21,7 +21,9 @@ def check_24h_window(session: Session, chat_session_id: int):
         select(Chat)
         .where(Chat.chat_session_id == chat_session_id)
         .where(
-            Chat.sender_role != Sender_Role_Enum.ASSISTANT,
+            # only chat from client/farmer will be calculated
```
[the remark this answers is not in the document]
Comment (wayangalihpratama) [reply backend/db/crud_chat.py:24]: Hi @jcbashdown , 
Yes, the client refer to the Farmer in our application.

File: backend/core/socketio_config.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (wayangalihpratama) [reply backend/core/socketio_config.py:248]: Yes correct,
I will fix that.

**Review by jcbashdown — APPROVED** [review]

File: backend/core/socketio_config.py [hunk 3]
```diff
@@ -144,91 +135,50 @@ async def save_chat_history(
...
         new_chat_status = Chat_Status_Enum.READ
+        final_message_body = message_body
 
-        # get message template lang
+        # Determine the message template language
         message_template_lang = generate_message_template_lang_by_phone_number(
-            phone_number=client_phone_number
+            client_phone_number
         )
         send_conversation_reconnect_template = check_24h_window(
```
[the remark this answers is not in the document]
Comment (wayangalihpratama) [reply backend/core/socketio_config.py]: Resolved

**Pull request merged**